.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -119,36 +104,22 @@ body:
     id: "nightwatch-version"
     attributes:
       label: "Nightwatch.js Version"
-      description: "The version of Nightwatch you use."
+      description: "The version of Nightwatch you use. `npx nightwatch --version`"
       placeholder: "2.3.0"
     validations:
       required: true
-  - type: "input"
-    id: "npm-version"
-    attributes:
-      label: "NPM Version"
-      description: "The version of NPM you use."
-      placeholder: "8.12.1"
-    validations:
-      required: true
   - type: "input"
     id: "node-version"
     attributes:
       label: "Node Version"
       description: "The version of Node you use."
       placeholder: "14.0.0"
-  - type: "input"
-    id: "yarn-version"
-    attributes:
-      label: "Yarn Version"
-      description: "The version of Yarn you use."
-      placeholder: "1.0.1"
```

Comment:
This, too (if a user uses yarn instead), can be clubbed with the above.

Reply:
These can be provided in additional information if required. What difference does yarn or npm does has on nightwatch executions? Only the node version actually has any impact.